test/acceptance/spec/20-details.js

```diff
@@ -4,11 +4,11 @@ var expectedRecord = require('../expected-record');
 var expectedRecords = require('../expected-records');
 
 describe('Details Page', () => {
-  const urlShouldContainDetails = () => {
-    it('returns a details page', () => {
-      browser.shouldBeOnDetailsPage();
-    });
-  };
+  const urlShouldContainDetails = () =>
+    it('returns a details page', () => browser.shouldBeOnDetailsPage());
```

Comment:
This could even be `it('returns a details page', browser.shouldBeOnDetailsPage);`. :-)

Reply:
Ha, nice try @daniel-ac-martin - but no, no you can't, because at the time `it` is called `browser` doesn't exist... it's not until the tests actually start to run (after they've been defined) that `browser` is created.